docs/index.md

```diff
@@ -1,51 +1,28 @@
-  <a class="button" href="./attend/firsttime"><span>Attend</span></a>
-  <a class="button" href="./organise/before/gettingstarted"><span>Organise</span></a>
-  <a class="button" href="./sponsor"><span>Sponsor</span></a>
-  <a class="button" href="./volunteer"><span>Volunteer</span></a>
-  <a class="button" href="./discord"><span>Chat</span></a>
-</div>
-
-## About
+---
+template: overrides/home.html
+---
 
-This is a Wiki for the UK Hackathon scene! Primarily focused on the various
-Student Hackathons ran at all the various Universities around the country.
+{% for season in extra.hackathon_seasons %}
 
-Here you can find information on what to expect when attending, event information
-and help with organising your own hackathon!
+{% if season.past == False %}
 
-We also have a Discord server, whether you're an experienced organiser or want
-to attend your first hackathon or something inbetween join us [here](https://hack.athon.uk/discord).
+## {{season.name}}
 
-## Contribute
+<div class="hack-list">
+    
+    {% for hackathon in season.hackathons %}
 
-You can find the Git Repository [here](https://github.com/Hackathons-UK/wiki).
-Feel free to send Pull Requests to update the content!
+        {% include 'events/card.md' %}
 
-# Upcoming Events
-This list showcases upcoming UK student-run hackathons and hackathon-associated events.
-Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/docs/index.md) to add your event!
+    {% endfor %}
 
-## Summer 2020
-
-|Hackathon        |Website|   Location         |No. of Hackers|Date|
-|-----------------|-------|--------------------|--------------|----|
-| STEAM AiHack | [hack.innovationfest.co.uk/AiHack/](https://hack.innovationfest.co.uk/AiHack/) | Birmingham (Online) | 100 | 27th-31st July 2020
+</div>
 
+{% endif %}
 
-## Autumn 2020
-|Hackathon        |Website|   Location         |No. of Hackers|Date|
-|-----------------|-------|--------------------|--------------|----|
-| HackTheMidlands 5.0 | [hackthemidlands.com](https://hackthemidlands.com) | Birmingham (Venue TBC) | 300 | 24th-25th October 2020 (TBC) |
-| R.U. Hacking? 2020 | [ruhacking.me](https://ruhacking.me) | Reading University | TBC | 24th-25th October 2020 |
-| Hack Brunel 2.0  | [hackbrunel.com](https://hackbrunel.com/) | Brunel University London | 150 | 31st October-1st November 2020 |
-| hackSheffield 6.0 | [hacksheffield.com](https://hacksheffield.com) | University of Sheffield | 200 | 7th-8th November 2020 |
-| DurHack 2020    | [durhack.com](https://durhack.com) | Durham University | 425 | 14th-15th November 2020 |
-| HackNotts 2020  | [hacknotts.com](https://www.hacknotts.com) | University of Nottingham | 200 | 28th-29th November 2020 |
+{% endfor %}
 
+Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/mkdocs.yml) to add your event! You can find the hackathons inside <code>hackathon_seasons:</code>
```

Comment:
Is this supposed to link to `mkdocs` file or just the repo?

Reply:
Should be the repo

Reply:
The mkdocks

Reply:
Why mkdocs file? If we're inviting people to contribute a link to the repo makes more sense imo

Reply:
If you wanted the repo am cool with that... but previously it linked to the file that had the hackathon data!

Reply:
The file has changed, that's all

Reply:
> If you wanted the repo am cool with that... but previously it linked to the file that had the hackathon data!

ahhh got it, makes sense